homeassistant/components/elevenlabs/tts.py

```diff
@@ -100,7 +100,7 @@ async def async_get_tts_audio(
         """Load tts audio file from the engine."""
         _LOGGER.debug("Getting TTS audio for %s", message)
         _LOGGER.debug("Options: %s", options)
-        voice_id = options[ATTR_VOICE]
+        voice_id = options.get(ATTR_VOICE, self._default_voice_id)
```

Comment:
Did we already always have voice set during setup?

Reply:
You mean default_voice_id? It is a string that is set in __init__, so before the Entity is passed to `async_add_entities`. I verified the test I added failed beforehand and works now and I can't manage to run into the #123597 issue with this branch

Reply:
I mean `ATTR_VOICE`

Reply:
I see. We're setting `CONF_VOICE `during setup, `ATTR_VOICE` is injected in the `prepare_text_to_speech` method of a PipelineRun, but only if the voice is set by the user. Since this isn't the case in the automation that raised the issue, we need the `.get()` here